soroban-env-host/src/auth.rs

```diff
@@ -828,7 +830,7 @@ impl AuthorizationTracker {
                 }
             }
         } else {
-            return Err(host.err_general("missing address to authenticate"));
+            return Err(ScHostAuthErrorCode::MissingAddress.into());
```

Comment:
Sorry, I forgot to leave a comment here. This would be an internal error and it shouldn't normally happen, so `err_general` should be used instead.

Reply:
Done